rmgpy/rmg/main.py

```diff
@@ -1356,6 +1356,15 @@ def makeSeedMech(self,firstTime=False):
             with open(os.path.join(filter_dir, 'family_map.yml'), 'w') as f:
                 yaml.dump(data=family_map, stream=f)
 
+            # Generate a file for restarting from a seed mechanism if this is not a restart job
```

Comment:
I'm not sure I understand, we don't want to generate a file to restart if the job we're running was ever restarted?

Reply:
We don't want to generate a restart file if the _current job_ is a restart job because there already exists a file with the restart block defined in it, no need to write the exact same file again.